pkg/virt-handler/non-root.go

```diff
@@ -137,15 +138,20 @@ func (d *VirtualMachineController) nonRootSetup(origVMI, vmi *v1.VirtualMachineI
 	return nil
 }
 
-func relabelFiles(newLabel string, files ...string) error {
+func relabelFiles(newLabel string, isPermissive bool, files ...string) error {
```

Comment:
we could move both of these relabelFiles functions to the selinux library

Reply:
Yes. Will do. :+1:

Reply:
Done.